app/js/Button.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export default class Button extends React.PureComponent {
+  constructor() {
+    super();
+  }
+
+  render() {
+    const classes = classNames(['glyphicon', this.props.className]);
+    const content = <span><i className={classes}></i> {this.props.label}</span>;
+
+    return (
+      <button className="btn btn-default btn-xs" onClick={this.props.clickHandler}>
+        {content}
+      </button>
+    );
+  }
+}
+
+Button.propTypes = {
+  label: PropTypes.string,
+  className: PropTypes.string,
+  clickHandler: PropTypes.func.isRequired,
```

Comment:
Since this passes through to `onClick`, I'd stick with the same name so the interface is familiar. Call it `onClick`.